### PR TITLE
Explain how users can define workspace deployment labels and annotations

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Explain how users can define workspace deployment labels and annotations

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18027

### Specify the version of the product this PR applies to.
7.21

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// configuring-a-workspace-using-a-devfile
+
+[id="allow-users-to-define-workspace-deployment-labels-and-annotations_{context}"]
```
Comment (yhontyk) [inline modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc]:
```suggestion
[id="allowing-users-to-define-workspace-deployment-labels-and-annotations_{context}"]
```

File: modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+= Allow users to define workspace deployment labels and annotations
```
Comment (yhontyk) [inline modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc]:
```suggestion
= Allowing users to define workspace deployment labels and annotations
```

Comment (yhontyk) [reply modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc]: Also, I suggest renaming the file itself (I don't think I can suggest it in GitHub through Suggestions =\)

File: modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+This section describes how to customize workspace deployment labels and annotation with help of factory parameters.
```
Comment (yhontyk) [inline modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc]:
```suggestion
This section describes how to customize workspace deployment labels and annotation using factory parameters.
```

File: modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+* A running instance of {prod}. To install an instance of {prod}, see xref:installation-guide:installing-che.adoc[].
+* A publicly accessible standalone `devfile.yaml` file. See xref:making-a-workspace-portable-using-a-devfile.adoc[] for detailed information about creating and using devfiles.
+
+.Procedure
+. Open the workspace by navigating to the following URL: `pass:c,a,q[{prod-url}/f?url=https://__<hostURL>__/devfile.yaml&workspaceDeploymentLabels=__<url_encoded_comma_separated_key_values>__&workspaceDeploymentAnnotations=__<url_encoded_comma_separated_key_values override>__]`
+
+.Example of overriding the `deployment labels` 
+
+Consider the following labels to add:
+
```
Comment (boczkowska) [inline modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc:21]:
```suggestion
+
```

File: modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc (hunk 2 above)
Comment (skabashnyuk) [reply modules/end-user-guide/partials/proc_allow-users-to-define-workspace-deployment-labels-and-annotations.adoc]: done

**Review by yhontyk — APPROVED** [review]
Thanks, approving.

**Pull request merged**